core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -71,64 +72,129 @@ object ConsumerGroupCommand {
       }
     } catch {
       case e: Throwable =>
-        println("Error while executing consumer group command " + e.getMessage)
-        println(Utils.stackTrace(e))
+        consumerGroupOutputWriter.printError("Executing consumer group command failed due to " + e.getMessage
+            + "\n" + Utils.stackTrace(e))
     } finally {
       consumerGroupService.close()
     }
   }
 
+  case class ConsumerGroupAssignment(group: String, topicOpt: Option[String], partitionOpt: Option[Int], offsetOpt: Option[Long],
+                                     lagOpt: Option[Long], memberIdOpt: Option[String], memberHostOpt: Option[String],
+                                     clientIdOpt: Option[String], logEndOffset: Option[Long])
+
+  trait OutputWriter {
+    def printError(msg: String)
```

Comment:
Actually I see that we print some context-specific error messages, so maybe we need to allow the message to come through. But perhaps we could pass the exception in an optional argument so that we can have a common place to log the stacktrace.